docs/source/en/chat_templating.md

```diff
@@ -343,6 +343,15 @@ tokenizer.push_to_hub("model_name")  # Upload your new template to the Hub!
 The method [`~PreTrainedTokenizer.apply_chat_template`] which uses your chat template is called by the [`ConversationalPipeline`] class, so 
 once you set the correct chat template, your model will automatically become compatible with [`ConversationalPipeline`].
 
+<Tip>
+If you're fine-tuning a model for chat, in addition to setting a chat template, you should probably add any new chat
```

Comment:
I'd remove `probably` here - it's sounds like the tip is unsure! 

```suggestion
If you're fine-tuning a model for chat, in addition to setting a chat template, you should add any new chat
```

Reply:
+1 here! But some models are trained without these tokens as special 😞

Reply:
Ah, OK. So maybe something like: `"If your fine-tuning a model for chat and it was trained with special tokens, you should..."`

Reply:
I think it's okay to leave it as-is - this is advice for people who are about to fine-tune a model, in which case there's still time for them to add the special tokens!

I agree it's too late once the model is already trained, of course